debian/tests/upgrade-between-snapshots

```diff
@@ -5,33 +5,36 @@ set -e
 chroot_dir=$AUTOPKGTEST_TMP/chroot
 start_date=20171001T000000Z
 end_date=20171210T000000Z
+chroot_exec="chroot $chroot_dir eatmydata"
 
-case "$(lsb_release -s -i)" in
-    Debian)
-        dpkg-buildpackage -us -uc 2>&1
-        debootstrap sid $chroot_dir "http://snapshot.debian.org/archive/debian/$start_date/ unstable main"
-        mount --bind /dev/pts $chroot_dir/dev/pts
-        sed -i 's/^deb /deb [check-valid-until=no] /' $chroot_dir/etc/apt/sources.list
-        chroot $chroot_dir apt-get update
-        # install u-u to pull in dependencies and mark python3.5-minimal
-        # because otherwise u-u autoremoves it while running python3.5 code
-        chroot $chroot_dir apt-get install -y unattended-upgrades python3.5-minimal mailutils
-        # add package set with many dependencies
-        chroot $chroot_dir apt-get install -y xfce4
-        chroot $chroot_dir apt-get clean
-        chroot $chroot_dir apt-mark showmanual > $chroot_dir/tmp/manual
-        cp ../unattended-upgrades_*.deb $chroot_dir/tmp/
-        chroot $chroot_dir bash -c 'dpkg -i /tmp/unattended-upgrades_*deb' 2>&1
-        # enable a few features to test
-        sed -i 's|//Unattended-Upgrade::Mail |Unattended-Upgrade::Mail |' $chroot_dir/etc/apt/apt.conf.d/50unattended-upgrades
-        sed -i "s/$start_date/$end_date/" $chroot_dir/etc/apt/sources.list
-        chroot $chroot_dir apt-get update
-        chroot $chroot_dir unattended-upgrade --verbose
-        echo "new packages marked as manually installed (should be none): "
-        chroot $chroot_dir apt-mark showmanual | diff $chroot_dir/tmp/manual -
-        chroot $chroot_dir perl -MMIME::QuotedPrint -pe '$_=MIME::QuotedPrint::decode($_);' /var/mail/mail
-        # cleanup
-        umount $chroot_dir/dev/pts
-        rm -rf $chroot_dir
-        ;;
-esac
+dpkg-buildpackage -us -uc 2>&1
+debootstrap --include eatmydata sid $chroot_dir "http://snapshot.debian.org/archive/debian/$start_date/ unstable main"
+mount --bind /dev/pts $chroot_dir/dev/pts
+sed -i 's/^deb /deb [check-valid-until=no] /' $chroot_dir/etc/apt/sources.list
+$chroot_exec apt-get update
+# install u-u to pull in dependencies and mark python3.5-minimal
+# because otherwise u-u autoremoves it while running python3.5 code
+$chroot_exec apt-get install -y unattended-upgrades python3.5-minimal mailutils
+# add package set with many dependencies
+$chroot_exec apt-get install -y xfce4
+$chroot_exec apt-get clean
+$chroot_exec apt-mark showmanual > $chroot_dir/tmp/manual
+
+# install package version just built
+cp ../unattended-upgrades_*.deb $chroot_dir/tmp/
+$chroot_exec bash -c 'dpkg -i /tmp/unattended-upgrades_*deb' 2>&1
```

Comment:
(nitpick) we could use `apt install -y /tmp/unattended-upgrades_*.deb` here now (unless we care about running this on old distros.

Reply:
I fixed that in an additional commit in the PR. It is indeed a bin nicer.